test/triggers/task_run_finished.js

```diff
@@ -113,17 +108,16 @@ describe('task run finished trigger', () => {
 
         expect(results.length).to.be.eql(3);
         expect(results[0].id).to.be.eql(runs.pop().id);
-        expect(results[0]).to.have.all.keys(Object.keys(TASK_RUN_SAMPLE));
+        expect(results[0]).to.have.all.keys(Object.keys(TASK_RUN_SAMPLE).concat(['isStatusMessageTerminal', 'statusMessage']));
```

Comment:
Maybe the `TASK_RUN_SAMPLE` should be extended?

Reply:
It is not possible as these attributes are there just a few runs not fo all of them.